invenio_files_rest/models.py

```diff
@@ -1161,6 +1164,37 @@ def copy(self, bucket=None, key=None):
 
         return new_ob
 
+    def clone(self):
+        db.session.expunge(self)
+        make_transient(self)
```

Comment:
done via python objects due to sqlalchemy not being flexible enough with insert values and instert from select 
another alternative would be doing it via pure sql queries but it does not really follow best practices